### PR TITLE
Return entry for stop on entry breakpoints

### PR DESCRIPTION
**Pull request by digeff**
Return entry for stop on entry breakpoints.
VS relies on the reason to auto-resume this breakpoint when needed

### CODE REVIEW

File: src/adapter/threads.ts [hunk 1]
```diff
@@ -845,11 +846,14 @@ export class Thread implements IVariableStoreDelegate {
         };
       default:
         if (event.hitBreakpoints && event.hitBreakpoints.length) {
+          const isStopOnEntry =
+            event.hitBreakpoints.length === 1 &&
+            this._delegate.entryBreakpointId === event.hitBreakpoints[0];
```
Comment (connor4312) [inline src/adapter/threads.ts]:
```suggestion
          const isStopOnEntry =
           this._delegate.entryBreakpointId &&
           event.hitBreakpoints.includes(this._delegate.entryBreakpointId);
```

in case we stop there for some other random reason

File: src/targets/node/nodeAttacherBase.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ export abstract class NodeAttacherBase<T extends AnyNodeConfiguration> extends N
       initialized: async () => {
         toDispose = this.onFirstInitialize(cdp, run);
         await toDispose;
+        return undefined;
```
Comment (connor4312) [inline src/targets/node/nodeAttacherBase.ts]: unrelated?

Comment (digeff) [reply src/targets/node/nodeAttacherBase.ts]: This is because I changed the signature of NodeTarget initialized method to:   
`initialized?(target: NodeTarget): Promise<string | undefined>;`

File: src/adapter/threads.ts (hunk 1 above)
Comment (digeff) [reply src/adapter/threads.ts]: My thinking was that if the user sets a breakpoint on this same place, then we don't want VS to auto-resume, so we want the breakpoint to take precedence and say "breakpoint" as a reason instead of "entry"

File: src/targets/node/nodeAttacherBase.ts (hunk 2 above)
Comment (digeff) [reply src/targets/node/nodeAttacherBase.ts]: This is because I changed the signature of NodeTarget initialized method to:   
`initialized?(target: NodeTarget): Promise<string | undefined>;`

File: src/adapter/threads.ts (hunk 1 above)
Comment (connor4312) [reply src/adapter/threads.ts]: ah, right, makes sense
